### DISCUSSION TITLE
Using Tempo behind reverse proxy with subpath

### DISCUSSION THREAD

**Discussion by rmarops** [grafana/grafana#74042] [Alerting]
Is it possible to serve tempo as described?  I get 404 not found (which is similar to Grafana when the GF_SERVER_SERVE_FROM_SUB_PATH and GF_ROOT_URL is not set). 

Example nginx subpath excerpt
```
upstream tempo_upstream {
    server tempo:3200;
}

upstream tempo_upstream_http {
    server tempo:4318;
}
server {
    location /tempo {
        proxy_pass             http://tempo_upstream;
        proxy_read_timeout     900;
        proxy_redirect off;
    }
    location /tempohttp {
        proxy_pass             http://tempo_upstream_http;
        proxy_read_timeout     900;
        proxy_redirect off;
    }
}
```

**Comment by rmarops** [comment]
Currently testing a rewrite rule
```
    location /tempohttp {
        rewrite                /tempohttp/(.*) /$1 break;
        proxy_pass             http://tempo_upstream_http;
        proxy_read_timeout     900;
        proxy_redirect off;
    }
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82576. Feel free to continue the discussion around this there. Thank you!
